src/vis_cpu/vis_cpu.py

```diff
@@ -108,21 +108,17 @@ def vis_cpu(
     beam_list : list of UVBeam, optional
         If specified, evaluate primary beam values directly using UVBeam
         objects instead of using pixelized beam maps (``bm_cube`` will be
-        ignored if ``beam_list`` is not ``None``).
+        ignored if ``beam_list`` is not ``None``). Note that if `polarized` is True,
+        these beams must be efield beams, and conversely they must be power beams
```

Comment:
Need another word or two between "conversely" and "they must". Maybe "conversely if `polarized` is False they must..."

Reply:
Thanks, done.